### PR TITLE
update shell syntax to support mac bash

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift-s2i/s2i-wildfly/issues/133

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng ptal

File: hack/build.sh [hunk 1]
```diff
@@ -88,7 +75,7 @@ for dir in ${dirs}; do
     docker_build_with_version Dockerfile
 #  fi
 
-  if [[ -v TEST_MODE ]]; then
+  if [ ! -z $TEST_MODE ]; then
```
Comment (csrwng) [inline hack/build.sh]: You can restore the double [[ (I think).
Also, I think that a more correct check would be:
```
if [[ -z "${TEST_MODE:-}" ]]; then
```

**Comment by bparees** [comment]
updated.
[test]

**Comment by bparees** [comment]
[test]

**Comment by csrwng** [comment]
Tested on Mac. LGTM

**Pull request merged**